### ISSUE TITLE
animatedshortfilms.net [Breakage] [FETL]

### ISSUE THREAD

**Issue by kulfoon** [ryanbr/fanboy-adblock#649]
### List the website(s) you're having issues:

`http://animatedshortfilms.net/`

### What happens?

Breakage - the page won't load - shadow overlay (after closing anti-adblock message).

### List Subscriptions you're using:

uBO default +
uBO annoyance +
Fanboy’s Enhanced Tracking List

### Your settings

- OS/version: Windows 7 SP1 64bit
- Browser/version: Firefox 63.0.1 64bit / Opera 56 64bit
- Adblock Extension/version: uBlock Origin version: 1.17.2

### Other details:

Screenshoots:

<details not-open>
<summary>screenshoot 1 - the anti-adblock message with or without enabling Fanboy’s Enhanced Tracking List</summary>

![ss_022](https://user-images.githubusercontent.com/43844449/48312077-b002b000-e5a9-11e8-8ae8-4ff58930cfe2.png)

</details>
.

<details not-open>
<summary>screenshoot 2 - after enabling Fanboy’s Enhanced Tracking List + after accepting the anti-adblock message</summary>

![ss_021](https://user-images.githubusercontent.com/43844449/48312078-b4c76400-e5a9-11e8-959a-1e91a5f54931.png)

</details>
.

<details not-open>
<summary>screenshoot 3 - after disabling Fanboy’s Enhanced Tracking List + after accepting the anti-adblock message</summary>

![ss_023](https://user-images.githubusercontent.com/43844449/48312080-b98c1800-e5a9-11e8-987c-7ac9c276c45d.png)

</details>

**Comment by THEtomaso** [comment]
I suspect that most (if not all) of the sites that you've reported today, simply needs the same fixes that I've posted here:
https://github.com/ryanbr/fanboy-adblock/issues/648#issuecomment-437672272
..and here:
https://github.com/ryanbr/fanboy-adblock/issues/650#issuecomment-437673568

If you feel up to it, you should test this from your end, and post the results.
This will save ryanbr a lot of work.

**Comment by kulfoon** [comment]
Not really, the issue is still here, but it won't fix anyway because it's the favicon thing again as per https://github.com/ryanbr/fanboy-adblock/issues/688#issuecomment-441432194.
